crates/egui-winit/Cargo.toml

```diff
@@ -62,14 +62,18 @@ egui = { workspace = true, default-features = false, features = ["log"] }
 
 ahash.workspace = true
 log.workspace = true
+
+# wayland-cursor shoud enable the fs feature as it otherwise doesn't compile.
+# Instead do this manually for now.
```

Comment:
Is there an relevant issue on `wayland-cursor` we can link to?

Reply:
Tried to understand this a bit better - it's an issue where wayland-cursor relied on wayland-backend to enable nix/fs. Somehow winit uses an older wayland-backend than wayland-cursor, leading to this problem.

This is moot in the future after https://github.com/Smithay/wayland-rs/commit/edd0f60d0baf09604553525c2636df5d6ba05d44, as rustix/shm enables rustix/fs as well.

Have a made a note of that here, quite a mess!